app/src/main/java/kth/jjve/xfran/models/Result.java

```diff
@@ -73,4 +66,16 @@ public String getComments() {
     public void setComments(String comments) {
         this.comments = comments;
     }
+
+    public boolean isScaled() {
+        return scaled;
+    }
+
+    public void setScaled(boolean scaled) {
+        this.scaled = scaled;
+    }
```

Comment:
I think you can put this method (and the ones above and below in one line, for ex

public void setScaled(boolean scaled) {this.scaled = scaled;}

makes the code a bit cleaner

Reply:
Hmm, actually you can collapse them on the left side of Android Studio UI, so I don't really see a benefit in writing in one line (because the automatic constructor of setters and getters does it in 3 lines, so I'd have to either rework it or stop using the automatic constructor). Do you know if there's a way to customize the automatic setter and getter to one line?

Reply:
I believe that android studio in some cases also hints you to put it on one line, so there might be a way. But yeah, in this case its a bit 50/50 between the community of coding people. Some people also deem this to be more readable than its one line alternative. I wouldn't make a problem about it